app/src/sharedTest/java/org/oppia/android/app/policies/PoliciesActivityTest.kt

```diff
@@ -149,6 +204,63 @@ class PoliciesActivityTest {
       }
     }
   }
+  @Test
```

Comment:
Newline before this test for consistency.

Reply:
Done.